pkg/actors/actors.go

```diff
@@ -1210,21 +1210,19 @@ func (a *actorsRuntime) CreateReminder(ctx context.Context, req *CreateReminderR
 			return err
 		}
 
-		// TODO: change the 3rd party library to take our format
-		jobSchedule := "@every " + req.Period
 		internalScheduleJobReq := &schedulerv1pb.ScheduleJobRequest{
 			Job: &runtimev1pb.Job{
-				Name:     a.actorsConfig.AppID + "||" + jobName,
-				Schedule: jobSchedule,
+				Name:     jobName,
+				Schedule: req.Period,
```

Comment:
the request period is the schedule?

Reply:
I was thinking of sending it as-is and letting the library handle period too. Now, thinking about it, I will put the "@Every" back. So, I can extend the format for "@Every x"